packages/eslint-plugin/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Fairly certain we will need to include `dependencies/`peerDependencies` here for what this package depends on.

Reply:
have added

Reply:
I believe all the parsers/plugins that we include also need to be included

Reply:
@ojkelly I believe this is still an issue.

Reply:
This file has a peer dep of eslint.

```
  "peerDependencies": {
    "eslint": "^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0"
  }
}
```

It doesn't depend on anything else?

Reply:
Ah, nevermind. Thought it relied on the `@typescript-eslint` stuff.